psiturk/experiment.py

```diff
@@ -263,16 +263,21 @@ def advertisement():
         # them to start the task again.
         raise ExperimentError('already_started_exp_mturk')
     elif status == COMPLETED:
-        # They've done the debriefing but perhaps haven't submitted the HIT
+        # They've finished the experiment but perhaps haven't submitted the HIT
         # yet.. Turn asignmentId into original assignment id before sending it
         # back to AMT
-        return render_template(
-            'thanks.html',
-            is_sandbox=(mode == "sandbox"),
-            hitid=hit_id,
-            assignmentid=assignment_id,
-            workerid=worker_id
-        )
+        use_psiturk_ad_server = CONFIG.getboolean('Shell Parameters', 'use_psiturk_ad_server')
+        if not use_psiturk_ad_server:
+            return render_template(
+                'thanks-mturksubmit.html',
+                using_sandbox=(mode == "sandbox"),
+                hitid=hit_id,
+                assignmentid=assignment_id,
+                workerid=worker_id
+            )
+        else: 
+            # then AMT isn't involved. Show them a thanks message and tell them to go away
+            return render_template( 'thanks.html' )
```

Comment:
Ahh, ok, so those variables were accidentally left in even though they weren't being used. Got it. Looks good then!